src/kvstore/src/main.rs

```diff
@@ -96,46 +135,155 @@ fn get(client: ManyClient<impl Identity>, key: &[u8], hex: bool) -> Result<(), M
     } else {
         let result: kvstore::GetReturns = minicbor::decode(&payload)
             .map_err(|e| ManyError::deserialization_error(e.to_string()))?;
-        let value = result.value.unwrap();
+        let value = result.value;
+
+        if let Some(value) = value {
+            if hex {
+                println!("{}", hex::encode(value.as_slice()));
+            } else {
+                std::io::Write::write_all(&mut std::io::stdout(), &value).unwrap();
+            }
+        } else {
+            println!("{:?}", value);
+        }
+
+        Ok(())
+    }
+}
+
+fn query(client: ManyClient<impl Identity>, key: &[u8]) -> Result<(), ManyError> {
+    let arguments = kvstore::QueryArgs {
+        key: key.to_vec().into(),
+    };
+
+    let payload = client.call_("kvstore.query", arguments)?;
+    if payload.is_empty() {
+        Err(ManyError::unexpected_empty_response())
+    } else {
+        let result: kvstore::QueryReturns = minicbor::decode(&payload)
+            .map_err(|e| ManyError::deserialization_error(e.to_string()))?;
 
-        if hex {
-            println!("{}", hex::encode(value.as_slice()));
+        let owner = if let Some(owner) = result.owner {
+            owner.to_string()
         } else {
-            std::io::Write::write_all(&mut std::io::stdout(), &value).unwrap();
+            "None".to_string()
+        };
+
+        if let Some(disabled) = result.disabled {
+            match disabled {
+                Either::Left(b) => {
+                    println!("{} ({})", owner, b)
```

Comment:
```suggestion
                    println!("{}, disabled", owner)
```

Reply:
I'll fix it but your suggestion is wrong since `b` can be `false`.